src/containers/layouts/articles/article3/article3.container.tsx

```diff
@@ -1,56 +1,77 @@
 import React from 'react';
 import { NavigationScreenProps } from 'react-navigation';
-import { Article } from '@src/core/model';
+import {
+  Article,
+  Comment, Profile,
+} from '@src/core/model';
 import { articles } from '@src/core/data/article';
+import {
+  profile1,
+  profile2,
+  profile3,
+  profile4,
+} from '@src/core/data/profile';
 import { Article3 } from './article3.component';
 
+const profiles: Profile[] = [
+  profile1,
+  profile2,
+  profile3,
+  profile4,
+];
+
 interface State {
   article: Article;
-  comments: string[];
-  currentComment: string;
+  currentCommentText: string;
 }
 
 export class Article3Container extends React.Component<NavigationScreenProps, State> {
 
   public state: State = {
     article: articles[0],
-    comments: [
-      'This is very useful information for me. Thanks for your article!',
-    ],
-    currentComment: '',
+    currentCommentText: '',
   };
 
-  private onCommentPress = () => {
+  private onCommentPress = (index: number) => {
 
   };
 
-  private onLikePress = () => {
+  private onLikePress = (index: number) => {
+
+  };
+
+  private onCommentItemPress = (index: number) => {
 
   };
 
   private onCommentTextChange = (text: string) => {
-    this.setState({ currentComment: text });
+    this.setState({ currentCommentText: text });
   };
 
   private onCommentSubmit = () => {
-    const comments: string[] = this.state.comments;
-    comments.push(this.state.currentComment);
+    const articleCopy: Article = this.state.article;
```

Comment:
nice but there is no need in implementing business logic

Reply:
Prefer not delete right now :)